editorials: Crypto Suites, hash alg sections, small others (#85)
openid/oid4vc-haip-sd-jwt-vc: openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

This specification uses the terms "Holder", "Issuer", "Verifier", and "Verifiable Credential" as defined in [@!I-D.ietf-oauth-sd-jwt-vc].

> Wallet Attestation:
> : Proof or assertion that a Wallet Instance is genuine, has not been tampered with, and can be trusted to securely handle Verifiable Credentials. The Wallet Attestation MUST be issued by an entity trusted for that purpose.
> 

Reviewer: ```suggestion
Wallet Attestation:
: Proof or assertion that a Wallet Instance is genuine, has not been tampered with, and can be trusted to securely handle Verifiable Credentials. It is issued by a trusted entity.

```

we cannot put normative statement in the definition. 
and if we want to add this, this definition should belong in VCI? cc @paulbastian @c2bo

Reviewer 2: We currently do not define Wallet Attestation the Terminology section of VCI, but in https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#walletattestation.

I guess it would make the most sense to expand the terminology section of VCI a bit (add key attestation and wallet attestation to the general terminology) and then expand the referenced Terminology in HAIP which currently says (on main):
> This specification uses the terms "Holder", "Issuer", "Verifier", "Wallet", and "Verifiable Credential" as defined in @!OIDF.OID4VCI] and [[OIDF.OID4VP](https://openid.github.io/oid4vc-haip/openid4vc-high-assurance-interoperability-profile-wg-draft.html#OIDF.OID4VP)].

* `aal`: OPTIONAL. JSON String asserting the authentication level of the Wallet and the key as asserted in the `cnf` claim.

> To obtain the Issuer's Public key for verification, Wallet Attestetions MUST support web-based key resolution as defined in Section 5 of [@!I-D.ietf-oauth-sd-jwt-vc]. The JOSE header `kid` MUST be used to identify the respective key.

Author: ```suggestion
To obtain the Issuer's Public key for verification, Wallet Attestations MUST support web-based key resolution as defined in Section 5 of [@!I-D.ietf-oauth-sd-jwt-vc]. The JOSE header `kid` MUST be used to identify the respective key.
```
